timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -295,40 +295,40 @@ void declare_context(py::module &m) {
         });
 }
 
-void declare_observable(py::module &m) {
+// void declare_observable(py::module &m) {
 
-    using Class = timemachine::Observable;
-    std::string pyclass_name = std::string("Observable");
-    py::class_<Class>(m, pyclass_name.c_str(), py::buffer_protocol(), py::dynamic_attr());
-}
+//     using Class = timemachine::Observable;
+//     std::string pyclass_name = std::string("Observable");
+//     py::class_<Class>(m, pyclass_name.c_str(), py::buffer_protocol(), py::dynamic_attr());
+// }
 
-void declare_avg_partial_u_partial_param(py::module &m) {
+// void declare_avg_partial_u_partial_param(py::module &m) {
 
-    using Class = timemachine::AvgPartialUPartialParam;
-    std::string pyclass_name = std::string("AvgPartialUPartialParam");
-    py::class_<Class, timemachine::Observable>(m, pyclass_name.c_str(), py::buffer_protocol(), py::dynamic_attr())
-        .def(py::init([](timemachine::BoundPotential *bp, int interval) {
-            return new timemachine::AvgPartialUPartialParam(bp, interval);
-        }))
-        .def(
-            "avg_du_dp",
-            [](timemachine::AvgPartialUPartialParam &obj) -> py::array_t<double, py::array::c_style> {
-                std::vector<int> shape = obj.shape();
-                py::array_t<double, py::array::c_style> buffer(shape);
+//     using Class = timemachine::AvgPartialUPartialParam;
+//     std::string pyclass_name = std::string("AvgPartialUPartialParam");
+//     py::class_<Class, timemachine::Observable>(m, pyclass_name.c_str(), py::buffer_protocol(), py::dynamic_attr())
+//         .def(py::init([](timemachine::BoundPotential *bp, int interval) {
+//             return new timemachine::AvgPartialUPartialParam(bp, interval);
+//         }))
+//         .def(
+//             "avg_du_dp",
+//             [](timemachine::AvgPartialUPartialParam &obj) -> py::array_t<double, py::array::c_style> {
+//                 std::vector<int> shape = obj.shape();
+//                 py::array_t<double, py::array::c_style> buffer(shape);
 
-                obj.avg_du_dp(buffer.mutable_data());
+//                 obj.avg_du_dp(buffer.mutable_data());
 
-                return buffer;
-            })
-        .def("std_du_dp", [](timemachine::AvgPartialUPartialParam &obj) -> py::array_t<double, py::array::c_style> {
-            std::vector<int> shape = obj.shape();
-            py::array_t<double, py::array::c_style> buffer(shape);
+//                 return buffer;
+//             })
+//         .def("std_du_dp", [](timemachine::AvgPartialUPartialParam &obj) -> py::array_t<double, py::array::c_style> {
+//             std::vector<int> shape = obj.shape();
+//             py::array_t<double, py::array::c_style> buffer(shape);
 
-            obj.std_du_dp(buffer.mutable_data());
+//             obj.std_du_dp(buffer.mutable_data());
```

Comment:
Good catch. The `Observable` and `AvgPartialUPartialParam` classes would've needed some updates in this PR, but @proteneer mentioned we might be able to deprecate them instead. I've added `pytest.mark.skip` decorators to the tests that depend on these in 16826f1, but I'm not sure these should all be deprecated.. Maybe we can update some of them to avoid needing the above instead?

Reply:
Just deprecate/comment them out entirely.

Reply:
Done in ac23026, 3a3610e, c8dac31, and d3f0ff1.

Reply:
Oh, looks like one more reference in tested code: https://github.com/proteneer/timemachine/blob/bdc42fb2cd9c32b76642ca5b581271a2b2dffb0d/tests/test_frame_filters.py#L11 . I'll get any references from untested code when updating `examples/`.

Reply:
Thanks! I removed this in beed645.